Packs/FeedMitreAttack/Integrations/FeedMitreAttack/MITRE.py

```diff
@@ -0,0 +1,396 @@
+                        "value": f"{indicator.get('value')}"
+                    })
+                    break
+            else:
+                if search.lower() in v.lower() and customFields.get('mitrename') not in [x.get('mitrename') for x in returnListMD]:
+                    returnListMD.append({
+                        'mitrename': customFields.get('mitrename'),
+                        'Name': f"[{customFields.get('mitrename', '')}]({urljoin(indicatorURL, indicator.get('id'))})",
+                    })
+                    entries.append({
+                        "id": f"{indicator.get('id')}",
+                        "value": f"{indicator.get('value')}"
+                    })
+                    break
+    returnListMD = sorted(returnListMD, key=lambda name: name['mitrename'])
+    returnListMD = [{"Name": x.get('Name')} for x in returnListMD]
+
+    if raw:
+        demisto.results({
+            "indicators": json.dumps()
+        })
+
+    demisto.results({
+        'Type': entryTypes['note'],
+        'Contents': returnListMD,
+        'ContentsFormat': formats['json'],
+        'HumanReadable': tableToMarkdown(f'MITRE Indicator search:', returnListMD),
+        'ReadableContentsFormat': formats['markdown'],
+        'EntryContext': {
+            'indicators(val.id && val.id == obj.id)': entries
+            }
+    })
+
+
+def reputation_command(client, args):
+    instanceName = demisto.integrationInstance()
+    indicator = args.get('indicator')
+    returnListMD = list()
+    entries = list()
+    allIndicators = list()
+    page = 0
+    size = 1000
+    rawData = demisto.searchIndicators(query=f'type:"{client.indicatorType}" value:{indicator}', page=page, size=size)
+    while(len(rawData.get('iocs', [])) > 0):
+        allIndicators.extend(rawData.get('iocs', []))
+        page += 1
+        rawData = demisto.searchIndicators(query=f'type:"{client.indicatorType}" value:{indicator}', page=page, size=size)
+    for indicator in allIndicators:
+        customFields = indicator.get('CustomFields')
+
+        # Build the markdown for the user
+        md = customFields.get('mitredescriptionmarkdown')
+        if customFields.get('mitreurls', None):
+            md += "\n_____\n## MITRE URLs\n" + customFields.get('mitreurls')
+        if customFields.get('mitrekillchainphases', None):
+            md += "\n_____\n## Kill Chain Phases\n" + customFields.get('mitrekillchainphases')
+        score = indicator.get('score')
+        value = indicator.get('value')
+        indicatorID = indicator.get('id')
+        ec = {
+            "DBotScore(val.Indicator && val.Indicator == obj.Indicator && val.Vendor && val.Vendor == obj.Vendor)": {
+                "Indicator": value,
+                "Type": client.indicatorType,
+                "Vendor": "MITRE ATT&CK",
+                "Score": score
+            },
+            "MITRE.ATT&CK(val.value && val.value = obj.value)": {
```

Comment:
@michalgold what should be the context standard for MITRE ATT&CK?

Reply:
for now there is no need for one, we have the defined indicator fields.

Reply:
Should I remove the context output?